source/solvers/vof.cc

```diff
@@ -2095,20 +2201,49 @@ VolumeOfFluid<dim>::apply_peeling_wetting(const unsigned int i_bc,
                 round(phase_lighter_fluid_q / n_q_points_face);
               double phase_values_cell =
                 phase_values_q / static_cast<double>(n_q_points_face);
+              double pressure_values_cell =
+                pressure_values_q / static_cast<double>(n_q_points_face);
+
+              // Check peeling condition on the pressure gradient
+              if (pressure_values_cell < this->pressure_monitored_avg and
```

Comment:
```suggestion
              if (pressure_values_cell < this->pressure_monitored_avg &&
```

Reply:
I do not have particular attachment to `&&` vs. `and`, but I want it to be consistent, at least in this solver. On other parts of the code, Bruno asked me to change to `and` so I changed it here also. Do you want me to revert back to `&&` everywhere?

Reply:
let's revert back to &&. "and" is not portable whereas && is.